### ISSUE TITLE
Panels should scroll when focus is moved out of view via tab/arrow

### ISSUE THREAD

**Issue by avvvvve** [musescore/MuseScore#25571]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Using arrows or tab in a scrollable panel will not scroll to the next focused thing.

Occurs in at least Properties, Instruments, and Mixer panels (not Palettes).

For example
1. Create a score with many instruments and open the mixer
2. Make your window small enough so the mixer and instruments panel both are scrollable.
3. Tab to the first instrument in the Instruments panel, then arrow down to move focus down. See that at the bottom of the panel, focus moves beyond the bounds but the panel doesn't scroll to keep focus in view.
4. Do the same for sounds in the mixer (using tab)

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/04afca92-b328-46d7-9d79-6cf278a3eaa7

### What is the latest version of MuseScore Studio where this issue is present?

Nightly

### Regression

I was unable to check

### Operating system

macOS 15.1

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by shubham-shinde-442** [comment]
Working on this.
